tests/acceptance/30_generic_methods/unsafe/sysctl_value.cf

```diff
@@ -0,0 +1,179 @@
+#######################################################
+#
+# Test sysctl value setting
```

Comment:
Also, what is the behaviour when using string based values and the min/max options?
It should be described in the Doc or/and in the tests

Reply:
Also, is there a bitwise limit on the min/max? like there is for the maths bundles

Reply:
is numerical when possible, else allphanumerical. I'm adapting the doc